include/cloysterhpc/answerfile.h

```diff
@@ -133,6 +134,26 @@ class AnswerFile {
 
     std::vector<std::shared_ptr<ITool>> m_tools;
 
+    struct AFPostfix {
+        struct SASL {
+            std::string username;
+            std::string password;
+        };
+        struct SMTP {
+            std::string server;
+            int port;
+            std::optional<SASL> sasl;
+            // Relay doesn't need to have a specific struct because it only
+            // needs 'server' and 'port'.
+        };
+        bool enabled = false;
+        std::vector<std::string> destination;
```

Comment:
I don't understand why this is a vector. What `m_destination` does?

Reply:
Ultimately it fills this field in the `main.cf` postfix file

```
inet_protocols = all
#proxy_interfaces =
#proxy_interfaces = 1.2.3.4
mydestination = $myhostname, localhost.$mydomain, localhost, $mydomain,
                {postfix_mydestination}
#mydestination = $myhostname, localhost.$mydomain, localhost, $mydomain
#mydestination = $myhostname, localhost.$mydomain, localhost, $mydomain,
#	mail.$mydomain, www.$mydomain, ftp.$mydomain
```